exporters/stats/stackdriver/src/test/java/io/opencensus/exporter/stats/stackdriver/StackdriverStatsExporterTest.java

```diff
@@ -165,8 +166,10 @@ public void registerViewAndExport() throws IOException {
       StackdriverStatsExporter.registerView(view);
 
       verify(mockStub, times(1)).createMetricDescriptorCallable();
-      // The timeout for verifying createTimeSeries needs to match the export interval of exporter.
-      verify(mockStub, timeout(1000).times(1)).createTimeSeriesCallable();
+      // createTimeSeriesCallable() should be called once per second. This test uses a timeout that
+      // is longer than the export interval in order to decrease the chance of failures due to
+      // timing.
+      verify(mockStub, timeout(2000).atLeast(1)).createTimeSeriesCallable();
```

Comment:
Can we make this test deterministic and avoid using sleep?

Reply:
Another option would be to use CountDownLatch to make the tests more parallel, but that would make the test more complicated and I'm not sure if it's worthwhile.

Reply:
I've left a TODO here to see if there's a better way to fix this.